open_seq2seq/optimizers/optimizers.py

```diff
@@ -24,9 +24,14 @@
 from __future__ import print_function
 from __future__ import unicode_literals
 
+import collections
 import six
 import tensorflow as tf
+from tensorflow.python.framework import ops
```

Comment:
I think it is a good practice to rewrite these imports in a usual TF way. For example, "ops.name_scope" should be just "tf.name_scope" and such. They are used in this way because inside the main TF code base that is the only way to do it, but later they add a bunch of convenient import shortcuts that we should use so that we (and everyone else) are not confused by these "unusual" imports.